web-api/openapi/back_office.yaml

```
@@ -446,7 +446,17 @@ components:
             Setting neither will default to `is_candidate = true`.
           type: boolean
         summarize:
-          description: Summarize the document snippets before further processing.
+          description: |-
+            Summarize the document snippets before further processing.
+
+            This is incompatible with `split`.
+          type: boolean
+          default: false
+        split:
+          description: |-
+            Split the input documents into multiple parts.
```

:art: to align with the `summarize` option
```suggestion
            Split the input documents into multiple parts before further processing.
```